### PR TITLE
Fix editorial issues in Data Model section.

### PR DESCRIPTION
**Pull request by msporny**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/pull/631.html" title="Last updated on Feb 16, 2021, 2:53 PM UTC (96103e0)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/631/b3550b1...96103e0.html" title="Last updated on Feb 16, 2021, 2:53 PM UTC (96103e0)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1134,20 +1134,20 @@ <h2>Relative DID URLs</h2>
   <section>
     <h1>Data Model</h1>
     <p>
-This specification defines a data model for <a>DID documents</a> that
-is capable of being serialized into multiple concrete <a>representations</a>. This
-section provides a high-level description of the data model, how different types
-of properties are expressed in the data model, and instructions for extending
-the data model.
+This specification defines a data model that can be used to express <a>DID
+documents</a> and other data structures that might then be serialized into
```
Comment (rhiaro) [inline index.html:1138]:
```suggestion
documents</a> and other data structures that can then be serialized into
```

File: index.html [hunk 2]
```diff
@@ -1134,20 +1134,20 @@ <h2>Relative DID URLs</h2>
...
     <p>
 A <a>DID document</a> consists of a <a data-cite="INFRA#maps">map</a> of <a
-data-cite="INFRA#map-entry">entries</a>, where each entry consists of
-an entry key/entry value pair. The data model contains at least two
-different classes of entries. The first class of entries are called
-properties, and are specified in section <a href="#core-properties"></a>.
-The second class are representation-specific entries, and
-are specified in section <a href="#representations"></a>.
+data-cite="INFRA#map-entry">entries</a>, where each entry consists of an entry
+key/entry value pair. The <a>DID document</a> data model contains at least two
```
Comment (rhiaro) [inline index.html:1146]:
```suggestion
data-cite="INFRA#map-entry">entries</a>, where each entry consists of a
key/value pair. The <a>DID document</a> data model contains at least two
```

File: index.html [hunk 3]
```diff
@@ -1162,10 +1162,10 @@ <h1>Data Model</h1>
     </figure>
 
     <p>
-All entry keys in the data model are <a
-data-cite="INFRA#strings">strings</a>. All entry values are expressed
-using one of the abstract data types in the table below while each
-<a>representation</a> specifies the concrete serialization format of each data type.
+All entry keys in the <a>DID Document</a> data model are <a
```
Comment (rhiaro) [inline index.html:1165]:
```suggestion
All entry keys in the <a>DID document</a> data model are <a
```

File: index.html [hunk 4]
```diff
@@ -1162,10 +1162,10 @@ <h1>Data Model</h1>
...
+data-cite="INFRA#strings">strings</a>. All entry values are expressed using one
+of the abstract data types in the table below while each <a>representation</a>
```
Comment (rhiaro) [inline index.html:1167]:
```suggestion
of the abstract data types in the table below, and each <a>representation</a>
```

File: index.html [hunk 5]
```diff
@@ -1277,26 +1293,26 @@ <h2>Extensibility</h2>
 For maximum interoperability, it is RECOMMENDED that extensions use the official
```
Comment (rhiaro) [inline index.html:1293]:
```suggestion
For maximum interoperability, it is RECOMMENDED that extensions use the
```

File: index.html [hunk 6]
```diff
@@ -1277,26 +1293,26 @@ <h2>Extensibility</h2>
...
 W3C DID Specification Registries mechanism [[?DID-SPEC-REGISTRIES]]. The use of
 this mechanism for new properties or other extensions is the only specified
-method that ensures that two different <a>representations</a> will be able to work
-together.
+method that ensures that two different <a>representations</a> will be able to
```
Comment (rhiaro) [inline index.html:1296]:
```suggestion
mechanism that ensures that two different <a>representations</a> will be able to
```

**Review by rhiaro — APPROVED** [review]

**Comment by msporny** [comment]
Editorial, at least one review, changes requested and made, no objections, merging.

**Comment by peacekeeper** [comment]
I disagree that this PR was "editorial", since it expands the role of the data model in a significant way, and I therefore think that this PR should be reverted while discussions about the data model are happening in https://github.com/w3c/did-core/pull/653, https://github.com/w3c/did-core/pull/654, https://github.com/w3c/did-core/pull/644, and other places.

**Comment by msporny** [comment]
> I disagree that this PR was "editorial", since it expands the role of the data model in a significant way, and I therefore think that this PR should be reverted while discussions about the data model are happening in #653, #654, #644, and other places.

There wasn't a single normative statement that was touched by this PR, so it was (and still is) editorial.

That said, based on the resolutions from the call today, I'll apply those changes in another PR to address the language that you're concerned with based on a tentative WG consensus position.

**Pull request merged**